iodata/formats/wfx.py

```diff
@@ -332,12 +332,26 @@ def load_one(lit: LineIterator) -> dict:
             'unrestricted', norba, norb - norba,
             data['mo_occ'], data['mo_coeff'], data['mo_energy'], None)
 
+    data_extra = {}
+    extra_list = ['keywords', 'model_name', 'num_perturbations', 'num_electrons',
```

Comment:
Not all the items listed in `extra_list` should be added `extra` item returned. Some of them like `nume_primitives` is already stored in `obasis` or `num_electrons`, `num_alpha_electron` and `num_beta_electron` that are already stored in `mo`.